Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -380,19 +380,24 @@ private void EmitTradeTick(Messages.Matched message)
             });
         }
 
+        /// <summary>
+        /// Creates websocket message subscriptions for the supplied symbols
+        /// </summary>
+        protected bool Subscribe(IEnumerable<Symbol> symbols, TickType tickType)
```

Comment:
I think that if `TickType` doesn't apply for this IDQH (for all similar cases) let's just not use it and drop it at the `subscriptionManager.Un/SubscribeImpl +=` level, will make the methods cleaner